sdk/core/perf/test/src/random_stream_test.cpp

```diff
@@ -4,37 +4,38 @@
 #include <gtest/gtest.h>
 
 #include <azure/perf/random_stream.hpp>
+#include <vector>
 
 TEST(circular_stream, basic)
 {
   size_t const totalSize = 1024 * 1024 * 3; // should give 5 loops
   size_t const chunk = 1024 * 1024;
   auto r_stream = Azure::Perf::RandomStream::Create(totalSize);
-  uint8_t buffer[chunk];
-  uint8_t buffer2[chunk];
```

Comment:
Out of curiosity, what was the bug here?

Reply:
MSVC complains or throw a warning saying that the method is allocating memory in the stack beyond the limit.  That's due to creating two arrays of 1M each, resulting in 2MB stack allocation.
Updating the arrays to a vector would use heap for the 2MB

Reply:
Nice!